lib/Mojo/Util.pm

```diff
@@ -282,8 +282,8 @@ sub tablify {
     }
   }
 
-  my $format = join '  ', map({"\%-${_}s"} @spec[0 .. $#spec - 1]), '%s';
-  return join '', map { sprintf "$format\n", @$_ } @$rows;
+  my @formats = (map({"\%-${_}s"} @spec[0 .. $#spec - 1]),'%s');
```

Comment:
Parentheses around the map are unnecessary. This should be run through perltidy.